i2c/lis3dh_i2c/lis3dh_i2c.c

```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include "pico/stdlib.h"
+#include "pico/binary_info.h"
+#include "hardware/i2c.h"
+
+/* Example code to talk to a LIS3DH Mini GPS module.
```

Comment:
Looks like the "Mini GPS module" here is probably a copy'n'paste typo from a different example?